companies/gyroscope.json

```diff
@@ -0,0 +1,26 @@
+{
+    "slug": "gyroscope",
+    "relevant-countries": [
+        "all"
+    ],
+    "categories": [
+        "health"
+    ],
+    "name": "Gyroscope Innovations, Inc.",
+    "address": "548 Market St #77142\nSan Francisco CA 94109\nUnited States of America",
+    "email": "support@gyrosco.pe",
+    "web": "https://gyrosco.pe/",
+    "sources": [
+        "https://gyrosco.pe/privacy/",
+        "https://businesssearch.sos.ca.gov/CBS/SearchResults?SearchType=NUMBER&SearchCriteria=C3716797"
+    ],
+    "required-elements": [
+        {
+            "desc": "email",
+            "type": "name",
+            "optional": false
+        }
+    ],
+    "suggested-transport-medium": "email",
```

Comment:
Since this is not a privacy-specific email, we can't set the suggested transport medium to email.